src/browser_action.js

```diff
@@ -223,6 +223,7 @@ browseraction.createQuickAddEvent_ = function(text, calendarId) {
         'Authorization': 'Bearer ' + authToken
       },
       success: function(response) {
+        addAlert('section', response.summary, response.htmlLink);
```

Comment:
`alert()` can be easily confused with the JavaScript `alert` function. How about calling this `showToast`?